homeassistant/helpers/entity_component.py

```diff
@@ -384,9 +388,12 @@ def _update_entity_states(self, now):
                     to_update.append(update_coro)
 
             for update_coro in to_update:
-                yield from update_coro
+                try:
+                    yield from update_coro
+                except Exception:  # pylint: disable=broad-except
+                    self.component.logger.exception(
+                        'Error while update entity from %s in %s',
+                        self.platform, self.component.domain)
```

Comment:
We use `logger.exception` which will automatically pick up the exception and print it 👍